index.html

```diff
@@ -35,21 +35,33 @@ <h2>Background</h2>
       </p>
       <p>
         In such a case, there is only one capture-source that makes sense - the current tab. An API
-        is required that only offers the current tab. Such an API is currently in the works, and its
-        tentative name is <code>getViewportMedia</code>. The main benefit of
-        <code>getViewportMedia</code> is that it will offer the user a confirmation-only dialog. The
-        drawbacks are the long time until this API is be standardized, implemented, and until the
-        security mechanisms it requires are widely adopted.
+        is required that only offers the current tab. Such an API is currently in the works - see
+        [[mediacapture-viewport]], which introduces
+        <a data-cite="mediacapture-viewport/#dom-mediadevices-getviewportmedia">getViewportMedia</a
+        >. The benefit of
+        <a data-cite="mediacapture-viewport/#dom-mediadevices-getviewportmedia">getViewportMedia</a>
+        is that it will offer the user a confirmation-only dialog. The drawbacks are the long time
+        until this API is be standardized, implemented, and until the security mechanisms it
+        requires become widely adopted.
       </p>
       <p>
-        We propose a simpler mechanism that can act as a stopgap measure until
-        <code>getViewportMedia</code> is standardized, implemented and adopted by the Web. This
-        mechanism is a dictionary member that indicates to the user agent that the application
-        prefers the current tab. The user agent then still shows the user a dialog that complies
-        with all of the requirements placed on
-        <a data-cite="SCREEN-CAPTURE#dom-mediadevices-getdisplaymedia">getDisplayMedia</a>, by
-        offering all possible sources to the user, but the current tab is presented as the most
-        prominent option.
+        We introduce a simpler mechanism that can act as a stopgap measure until
+        <a data-cite="mediacapture-viewport/#dom-mediadevices-getviewportmedia">getViewportMedia</a>
+        is standardized, implemented and adopted by the Web. This mechanism is a dictionary member
+        that indicates to the user agent that the application prefers the current tab. The user
+        agent can then choose between:
+      </p>
+      <ul>
+        <li>
+          Showing the user a dialog that complies with all of the requirements placed on
+          <a data-cite="SCREEN-CAPTURE#dom-mediadevices-getdisplaymedia">getDisplayMedia</a>,
+          offering all possible sources to the user, but the current tab is presented as the most
+          prominent option.
+        </li>
+        <li>Showing the user a dialog that only offers the option of capturing the current tab.</li>
+      </ul>
```

Comment:
I think you can simplify this to "The user agent can then take that preference into consideration when representing the request to the user."

Reply:
Done.